src/components/popover-container/popover-container.component.tsx

```diff
@@ -71,22 +161,24 @@ const PopoverContainer = ({
   const renderOpenComponentProps = {
     tabIndex: isOpen ? -1 : 0,
     isOpen,
-    dataElement: "popover-container-open-component",
+    "data-element": "popover-container-open-component",
     onClick: handleOpenButtonClick,
     ref: openButtonRef,
-    ariaLabel: openButtonAriaLabel || title,
+    "aria-label": openButtonAriaLabel || title,
     id: isOpen ? undefined : popoverContainerId,
   };
 
   const renderCloseComponentProps = {
-    dataElement: "popover-container-close-component",
+    "data-element": "popover-container-close-component",
     tabIndex: 0,
     onClick: handleCloseButtonClick,
     ref: closeButtonRef,
-    ariaLabel: closeButtonAriaLabel,
+    "aria-label": closeButtonAriaLabel,
   };
 
-  const handleClickAway = (e) => {
+  // TODO: Assign proper type after ClickAwayWrapper has been refactored
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const handleClickAway = (e: any) => {
```

Comment:
See related comment above
```suggestion
  const handleClickAway = (e: CustomEvent) => {
```